### PR TITLE
Report language translation completeness on language_dump.py

### PR DESCRIPTION
**Pull request by tupaschoal**
~*This is in draft while #77 is not merged, as it builds on top of it*~

This reports the translation completeness when extracting a language with `language_dump.py`:
It's in the form of:
```pt-BR: 99.96% completeness```

### CODE REVIEW

File: language_dump.py [hunk 1]
```diff
@@ -63,10 +69,18 @@ def dump_translations():
...
-        extract_translations(language_to_extract, fallback_language, verbose,\
-                             filename, strings_by_object)
+        obj_translations, ref_obj_translation =\
+            extract_translations(language_to_extract, fallback_language, verbose,\
+                                 filename, strings_by_object)
+        reference_str_count += ref_obj_translation
+        translated_str_count += obj_translations
+
+    print('{}: {}% completeness'.format(language_to_extract,
+                                        round(100*translated_str_count/reference_str_count, 2)))
```
Comment (AaronVanGeffen) [inline language_dump.py:83]: The formatting here is a little awkward. My suggestion would be to combine local assignments with f-strings again.

Comment (tupaschoal) [reply language_dump.py]: Right, I forgot to update when rebasing, done now.

File: language_dump.py [hunk 2]
```diff
@@ -63,10 +69,18 @@ def dump_translations():
...
     for filename in glob.iglob(args.objects + '/**/*.json', recursive=True):
-        extract_translations(language_to_extract, fallback_language, verbose,\
-                             filename, strings_by_object)
+        obj_translations, ref_obj_translation =\
+            extract_translations(language_to_extract, fallback_language, verbose,\
+                                 filename, strings_by_object)
+        reference_str_count += ref_obj_translation
+        translated_str_count += obj_translations
+
+    translation_progress = round(100*translated_str_count/reference_str_count, 2)
```
Comment (AaronVanGeffen) [inline language_dump.py:82]:
```suggestion
    translation_progress = round(100 * translated_str_count / reference_str_count, 2)
```

Are you using pylint? I'd imagine this kind of thing would be covered by it.

Comment (tupaschoal) [reply language_dump.py:82]: I'm it actually doesn't suggest this.

**Pull request merged**